tinygrad/codegen/uops.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 from typing import Iterator, Optional, Tuple, Any, Dict, List, DefaultDict, Set, Callable, Union, cast, TypeVar
 import functools, itertools, heapq
+from math import isnan
```

Comment:
just `import functools, itertools, heapq, math`. The point is to have `math` be in the same line as other std imports